code/Commenting.php

```diff
@@ -40,6 +40,8 @@ class Commenting {
 		'html_allowed' => false, // allow for sanitized HTML in comments
 		'html_allowed_elements' => array('a', 'img', 'i', 'b'),
```

Comment:
I think this whole config system is antiquated and should be using the current `Config` api. Not a problem with your PR just noting it. :)